docs/sql-migration-guide-upgrade.md

```diff
@@ -159,6 +159,32 @@ license: |
 
   - Since Spark 3.0, Dataset query fails if it contains ambiguous column reference that is caused by self join. A typical example: `val df1 = ...; val df2 = df1.filter(...);`, then `df1.join(df2, df1("a") > df2("a"))` returns an empty result which is quite confusing. This is because Spark cannot resolve Dataset column references that point to tables being self joined, and `df1("a")` is exactly the same as `df2("a")` in Spark. To restore the behavior before Spark 3.0, you can set `spark.sql.analyzer.failAmbiguousSelfJoin` to `false`.
 
+  - Since Spark 3.0, we pad decimal numbers with trailing zeros to the scale of the column for Hive result, for example:
```

Comment:
To me, it looks not clear what Hive result is.

Reply:
Which one do you think is better?
![image](https://user-images.githubusercontent.com/5399861/63572295-23176900-c5b5-11e9-9618-fbfe22c08270.png)

---

![image](https://user-images.githubusercontent.com/5399861/63573083-011ee600-c5b7-11e9-8831-0815e25fa983.png)